iceoryx_posh/include/iceoryx_posh/version/compatibility_check_level.hpp

```diff
@@ -28,6 +30,32 @@ enum class CompatibilityCheckLevel
     BUILD_DATE
 };
 
+inline iox::log::LogStream& operator<<(iox::log::LogStream& logstream, const version::CompatibilityCheckLevel& level)
+{
+    switch (level)
+    {
+    case CompatibilityCheckLevel::OFF:
+        logstream << "OFF";
```

Comment:
The translation is exactly the same then the MonitoringMode translation. This could lead to confusion since the developer maybe assumes that the translation is detailed enough and does not provide some context which OFF is meant.

Could you please therefore translate this to `CompatibilityCheckLevel::OFF` and the values below with the same logic (EnumName::VALUE)

Reply:
Fixed.

Reply:
@mossmaurice either you forgot to commit your fix or my github page is showing me for some reasons the old state :/

Reply:
It's showing the **Outdated** version as stated here in the first line ;-)